src/solver/conv_hip_implicit_gemm_fwd_v4r4_xdlops.cpp

```diff
@@ -152,8 +152,11 @@ void PerformanceImplicitGemmForwardV4R4Xdlops::EuristicInit(const ConvolutionCon
                         break;
 
                     all_visited = true;
+                    break;
```

Comment:
However, logically, it's possible to run here, "all_visited = true", and if that would be, it will be an endless loop, here.

Reply:
This do_while loop is not straightforward, I also had hard time understand it when I modifed it. But I think because of ```while(false)``` below, this is actually not a loop, it's just a single pass, I guess the reason ```do while``` is used here is to use the ```break``` inside.

Reply:
oh, it's 'while(false)', so there will be no dead loop here.
ok, I also think no need to change it here.